ci/jenkins/test-vmc.sh

```diff
@@ -502,6 +502,7 @@ function run_e2e {
     ${SCP_WITH_ANTREA_CI_KEY} $GIT_CHECKOUT_DIR/jenkins/out/kubeconfig capv@${control_plane_ip}:~/.kube/config
 
     set +e
+    kubectl taint nodes -l node-role.kubernetes.io/control-plane!=true node.cluster.x-k8s.io/uninitialized-
```

Comment:
1. What's the intention of the selector "node-role.kubernetes.io/control-plane!=true"? The control-plane node has the key but not the value, so it will still select all nodes.
2. Why the uninitialized taint is on the worker nodes? When is it supposed to be deleted?

Reply:
CAPV by default needs cloud provider to mark the worker nodes initialized (by removing this taint). Our lab testbed doesn't integrate with cloud provider, so we can remove this taints via script. This was my previous investigation for old CAPV release. It's better to find out whether this taint can be removed automatically by CAPV.

Reply:
Updated the command to only pick the nodes without the "node-role.kubernetes.io/control-plane" key.

Reply:
OK, we can merge it as is to unblock tests, and investigate the workflow can be optimized later.

Reply:
Yes, this time I hope we can quickly recover the testbed by updating the script. In the long term, we can investigate how to update CAPV cluster yaml to removed it automatically. @edwardbadboy